tests/model_serving/model_runtime/conftest.py

```
@@ -0,0 +1,48 @@
+            )
+
+            LOGGER.info(f"Verifying if {operator_name} is installed")
+            for csv in csvs:
+                if csv.name.startswith(operator_name):
+                    if csv.status == csv.Status.SUCCEEDED:
+                        break
+
+                    else:
+                        missing_operators.append(
+                            f"Operator {operator_name} is installed but CSV is not in {csv.Status.SUCCEEDED} state"
+                        )
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "break", remove the "else" and de-indent the code inside it

(R1723)